package/lib-openvino/scripts.linux/install.sh

```diff
@@ -116,15 +116,18 @@ echo "Copying the include files to '${INC_DIR}' ..."
 cp -r ${OPENVINO_DIR}/inference-engine/include/* ${INC_DIR}
 exit_if_error "copying the include files failed"
 
+if [ "${PACKAGE_GIT_CHECKOUT}" == "2019_R3.1" ] || [ "${PACKAGE_GIT_CHECKOUT}" == "2019_R3" ] || [ "${PACKAGE_GIT_CHECKOUT}" == "pre-release" ]; then
```

Comment:
Does this mean there's no need for the calibration tool for any 2020 releases?

Reply:
The mechanism for calibration has changed. How this is handled will need to be looked into.